scripts/install_protoc.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright 2024 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu -o pipefail
+
+source "$(dirname $0)/vet-common.sh"
+
+# Perform installation of protoc from source based on OS.
+
+PROTOC_VERSION="25.2"
+
+# Function to download pre-built binaries for Linux with
+# ARCH as $1, OS as $2, and WORKDIR as $3 arguments.
+download_binary() {
+  # Check if protoc is already available
+  if command -v protoc &> /dev/null; then
+      if installed_version=$(protoc --version | cut -d' ' -f2 2>/dev/null); then
```

Comment:
inconsistent with other vars.. s/installed_version/INSTALLED_VERSION/ ?

Reply:
the naming style is not consistent with the other variables you have introduced. `installed_version` is in small case.